18F/identity-design-system
# Fix watch command for documentation build

## 🛠 Summary of changes

Updates `package.json` `scripts` to resolve an issue where `npm start` won't rebuild stylesheets on changes.

The issue is that `watch:docs:css` runs `build:docs:css`, which in turn runs `npm run build:css`. The `--` is required to pass flags via NPM commands, but it only applies to the initial `build:docs:css`, at which point it gets dropped. Suffixing `build:docs:css`'s command with `--` ensures those flags are carried along to `npm run build:css`.

Alternatively, it could probably be simpler to just call `build:css` directly, but this maintains some symmetry between `build:docs:css` and `watch:docs:css` and leaves open the possibility of documentation-specific extras. I'm not strongly attached though.

## 📜 Testing Plan

1. Run `npm start`
2. Go to http://localhost:3000 and view a component preview
3. Make a change to the stylesheet for that component that should have a visual effect
4. Refresh the page after a few seconds
5. Observe the stylesheet change takes effect
